INFO     [🔧TEST CONFIG]:conftest.py:84 Loading test configuration
INFO     [🔧TEST CONFIG]:conftest.py:371 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:413 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🔧TEST CONFIG]:conftest.py:90 Creating data manager
INFO     [🔧TEST CONFIG]:conftest.py:116 Creating users manager
INFO     [🔧TEST CONFIG]:conftest.py:102 Creating API helper
INFO     [🔧TEST CONFIG]:conftest.py:110 Creating Apolo CLI instance
INFO     [🔧TEST CONFIG]:conftest.py:198 Continue using UserData(email='regression-36g2p1z3vl@apolo.us', username='regression-36g2p1z3vl', password='iLNm^TkZ53xs') as main test user...
INFO     [👤UsersManager]:users_manager.py:73 Setting main user to UserData(email='regression-36g2p1z3vl@apolo.us', username='regression-36g2p1z3vl', password='iLNm^TkZ53xs')
INFO     [🔧TEST CONFIG]:conftest.py:228 Continue using UserData(email='regression-ln52hckrtj@apolo.us', username='regression-ln52hckrtj', password='d$v1RPzs8tCF') as second test user...
INFO     [👤UsersManager]:users_manager.py:82 Setting second user to UserData(email='regression-ln52hckrtj@apolo.us', username='regression-ln52hckrtj', password='d$v1RPzs8tCF')
INFO     [🔧TEST CONFIG]:conftest.py:233 Continue using UserData(email='regression-edy71rhsxa@apolo.us', username='regression-edy71rhsxa', password='1$iAq6K!OT$a') as third test user...
INFO     [👤UsersManager]:users_manager.py:91 Setting third user to UserData(email='regression-edy71rhsxa@apolo.us', username='regression-edy71rhsxa', password='1$iAq6K!OT$a')
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify Apolo CLI client is installed
INFO     [🖥apolo_CLI]:apolo_runner.py:25 check CLI version. Running command via cli:

apolo --version

INFO     [🖥apolo_CLI]:apolo_runner.py:86 check CLI version succeeded:

Apolo Platform Client 25.8.1

INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify Apolo CLI client is installed
INFO     [📘TEST_INFO]:reporting.py:86 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:87 📝 TEST started: User with organization logs in with auth token via CLI
INFO     [📘TEST_INFO]:reporting.py:88 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Login via UI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click login button
INFO     [🎭 Playwright]:base_page.py:16 <AuthPage>: Click log in button
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click login button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_login_button_success_13d19a.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter email
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter regression-36g2p1z3vl@apolo.us email
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter email
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_email_success_011366.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter password
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter ********* password
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter password
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_password_success_688405.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click Continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Click continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Timeout error for spinner
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click Continue button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_continue_button_success_a2c6be.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify that Welcome new user page displayed
INFO     [🎭 Playwright]:base_page.py:16 <WelcomeNewUserPage>: Check if page loaded
WARNING  [🔧TEST CONFIG]:conftest.py:413 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": true, "orgs": [], "clusters": [], "projects": [], "admin_url": "https://api.dev.apolo.us/apis/admin/v1", "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
WARNING  [🔧TEST CONFIG]:conftest.py:413 [HTTP 200] GET https://api.dev.apolo.us/api/v1/users/regression-36g2p1z3vl/permissions?uri=blob:
Request Body:
<not available>
Response Body (first 1000 chars):
[]
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Verify that Welcome new user page displayed
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/verify_ui_page_displayed_fail_e4bd07.png
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Login via UI
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_login_fail_01cfcd.png
ERROR    [📘TEST_INFO]:reporting.py:95 ❌ TEST failed: User with organization logs in with auth token via CLI
Exception: [AssertionError] during step: Login via UI
Message: [AssertionError] during step: Verify that Welcome new user page displayed
Message: Locator expected to be visible
Actual value: <element(s) not found> 
Call log:
  - Expect "to_be_visible" with timeout 5000ms
  - waiting for locator("h3.truncate.text-h3").filter(has_text="Welcome, regression-36g2p1z3vl@apolo.us")

INFO     [📘TEST_INFO]:reporting.py:98 ------------------------------------------------------------
INFO     [🌐API_helper]:api_helper.py:45 GET https://api.dev.apolo.us/apis/admin/v1/orgs
WARNING  [🔧TEST CONFIG]:conftest.py:413 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/invites/incoming
Request Body:
<not available>
Response Body (first 1000 chars):
[]
WARNING  [🔧TEST CONFIG]:conftest.py:413 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-36g2p1z3vl?include=orgs
Request Body:
<not available>
Response Body (first 1000 chars):
{"name": "regression-36g2p1z3vl", "email": "regression-36g2p1z3vl@apolo.us", "first_name": null, "last_name": null, "created_at": "2025-09-10T01:31:57.112055+00:00", "terms_accepted": true, "orgs": []}
WARNING  [🔧TEST CONFIG]:conftest.py:413 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-36g2p1z3vl?include=clusters&include=system_role
Request Body:
<not available>
Response Body (first 1000 chars):
{"name": "regression-36g2p1z3vl", "email": "regression-36g2p1z3vl@apolo.us", "first_name": null, "last_name": null, "created_at": "2025-09-10T01:31:57.112055+00:00", "terms_accepted": true, "clusters": []}
INFO     [🌐API_helper]:api_helper.py:137 Status: 200. Response: []
INFO     [🔧TEST CONFIG]:conftest.py:284 Cleaning up 0 organisations
INFO     [🔧TEST CONFIG]:conftest.py:314 Closing all browser sessions
INFO     [🔧TEST CONFIG]:conftest.py:320 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:330 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:335 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:340 Browser cleanup finished